ORM/LoggingHydratorTrait.php

```diff
@@ -13,54 +13,103 @@
 
 namespace Debesha\DoctrineProfileExtraBundle\ORM;
 
-use Countable;
 use Doctrine\DBAL\Result;
+use Doctrine\ORM\Internal\Hydration\AbstractHydrator;
 use Doctrine\ORM\Internal\Hydration\ArrayHydrator;
 use Doctrine\ORM\Internal\Hydration\ObjectHydrator;
 use Doctrine\ORM\Internal\Hydration\ScalarHydrator;
 use Doctrine\ORM\Internal\Hydration\SimpleObjectHydrator;
 use Doctrine\ORM\Internal\Hydration\SingleScalarHydrator;
 use Doctrine\ORM\Query\ResultSetMapping;
 
-trait LoggingHydratorTrait
-{
-    /**
-     * Hydrates all rows returned by the passed statement instance at once.
-     *
-     * @param Result|ResultStatement $stmt
-     * @param ResultSetMapping       $resultSetMapping
-     * @psalm-param array<string, string> $hints
-     *
-     * @return mixed[]
-     */
-    public function hydrateAll(/* Result */$stmt, /*ResultSetMapping*/ $resultSetMapping, /*array*/ $hints = [])/*: Countable|array*/
+if (property_exists(AbstractHydrator::class, '_em')) {
```

Comment:
This redeclaration is needed because the signature of hydrateAll changed. This causes issues in ORM2 where the typing is less strict.